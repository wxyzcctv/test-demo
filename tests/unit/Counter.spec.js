import { mount } from "@vue/test-utils";
import Counter from "@/components/Counter.vue";

describe("Counter.vue", () => {
  it("renders props.msg when passed", () => {
    const wrapper = mount(Counter);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
