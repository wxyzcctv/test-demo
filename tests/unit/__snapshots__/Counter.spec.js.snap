// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter.vue renders props.msg when passed 1`] = `<div><span>0</span> <button>count++</button></div>`;
